src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -55,7 +55,7 @@ class CallActor(call: Call, locallyQualifiedInputs: CallInputs, backend: Backend
   when(CallNotStarted) {
     case Event(Start, _) =>
       val backendInputs = backend.adjustInputPaths(call, locallyQualifiedInputs)
-      call.instantiateCommandLine(backendInputs) match {
+      call.instantiateCommandLine(backendInputs, backend.engineFunctions(backend.setupCallEnvironment(call, workflowDescriptor))) match {
```

Comment:
I think that this would be better if we captured the notion of a fully setup backend as a type, created an instance of that type in either the props method or an apply method and then passed *that* in instead of the actual backend to the call actor.